charts/dapr/charts/dapr_operator/templates/dapr_operator_deployment.yaml

```diff
@@ -175,6 +175,14 @@ spec:
         - "{{ .Values.global.prometheus.port }}"
 {{- else }}
         - "--enable-metrics=false"
+{{- end }}
+{{- with .Values.global.issuerFilenames }}
+        - --issuer-ca-filename={{ .ca }}
```

Comment:
Please keep flags and values in separate lines and make sure to quote the values. This is so if the values contain spaces, they don't break everything. Exampke:

```yaml
- "--issuer-ca-filename"
- "{{ .ca }}"
```